terminus-ui/cohort-date-range/src/cohort-date-range.component.ts

```diff
@@ -0,0 +1,152 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+  ViewEncapsulation,
+} from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+
+/**
+ * Represents date cohort object that passed in
+ */
+export interface TsDateCohort {
+  display: string;
+  range: {
+    startDate: string | Date;
+    endDate: string | Date;
+  };
+}
+
+/**
+ * Event object emitted by TsCohortDateRange when there is date range change
```

Comment:
When referencing a component/class in a JSDoc comment, we should link to it like: `blah {@link TsCohortDateRange} blah`. Our docs generator (and webstorm) will convert that into an actual link for easy reference.